meta-resin-common/recipes-support/resin-vars/resin-vars/resin-vars

```diff
@@ -34,9 +34,17 @@ while [[ $# > 0 ]]; do
     shift
 done
 
+# Retrive resin-boot mountpoint
+BOOT_MOUNTPOINT="$(grep $(readlink -f /dev/disk/by-label/resin-boot) /proc/mounts | cut -d ' ' -f 2)"
+
+if [ -z "$BOOT_MOUNTPOINT" ]; then
+    echo "[ERROR] : Could not determine mount point for resin-boot partition"
+    exit 1
```

Comment:
Okay, I was just suggesting this component is not responsible with whether resin-boot has been mounted or not. This component should just be responsible with setting the env vars from a file config.json, wherever that may be.
And I was saying that if CONFIG_PATH is not set and it gets set to the default value and at that point the BOOT_MOUNTPOINT is empty, only then it should exit.
Just my view on this